packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1558,6 +1560,41 @@ describe('RpcContractProvider test', () => {
     });
   });
 
+  describe('increasePaidStorage', () => {
+    it('should produce an increasePaidStorage operation', async (done) => {
```

Comment:
could you add a test :
should produce a reveal and an increasePaidStorage operation

Reply:
looks like the current test already does both, I will update the test name then

Reply:
we should have a test that include a reveal if needed and on that doesn't

Reply:
I forgot to mention about this case too: validate that the user-specified fees will be taken into account when specified